qa/tasks/cephfs/test_scrub.py

```diff
@@ -176,3 +177,12 @@ def test_scrub_backtrace(self):
 
     def test_scrub_dup_inode(self):
         self._scrub(DupInodeWorkload(self, self.fs, self.mount_a))
+
+    def test_mdsdir_scrub_backtrace(self):
+        damage_count = self._get_damage_count()
+        self.assertNotIn("MDS_DAMAGE", self.mds_cluster.mon_manager.get_mon_health()['checks'])
+
+        self.fs.run_scrub(["start", "~mdsdir", "recursive"])
+        self.fs.wait_until_scrub_complete(sleep=10, timeout=60)
```

Comment:
avoid specifying explicit sleep when using `wait_until_scrub_complete()`
the function `wait_until_scrub_complete()` uses sleep inside the function's implementation
please use the `"scrub_tag"` returned by the `run_scrub()` function and pass it to the `wait_until_scrub_complete()` function

e.g. check the usage in `qa/tasks/cephfs/test_forward_scrub.py`

Reply:
I saw some instances passing `sleep` expilicitly, that's why. I have updated as per your suggestion.